youtube_dl/extractor/webofstories.py

```diff
@@ -98,3 +100,26 @@ def _real_extract(self, url):
             'description': description,
             'duration': duration,
         }
+
+
+class WebOfStoriesPlaylistIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?webofstories\.com/playAll/(?P<id>[^/]+)'
+    _TESTS = []
```

Comment:
Item 6 of [this instruction](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site).